src/Elastic.Apm/DiagnosticListeners/KnownListners.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to Elasticsearch B.V under
+// one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+using System.Collections.Generic;
+
+namespace Elastic.Apm.DiagnosticListeners
+{
+	internal static class KnownListeners
+	{
+		public const string MicrosoftAspNetCoreHostingHttpRequestIn = "Microsoft.AspNetCore.Hosting.HttpRequestIn";
+		public const string SystemNetHttpHttpRequestOut = "System.Net.Http.HttpRequestOut";
+		public const string SystemNetHttpDesktopHttpRequestOut = "System.Net.Http.Desktop.HttpRequestOut";
+		public const string ApmTransactionActivityName = "ElasticApm.Transaction";
```

Comment:
Would it make sense to keep these as constants in their respective types, and just collate the set of them here?